server/routes/varyingLicences/handlers/viewActiveLicence.ts

```diff
@@ -18,10 +22,21 @@ export default class ViewActiveLicenceRoutes {
     const { conditionsWithUploads, additionalConditions } = this.conditionService.additionalConditionsCollection(
       licence.additionalLicenceConditions
     )
+    parentOrSelfAdditionalConditions = additionalConditions
```

Comment:
should this be filtered to only show the AP conditions belonging to the parent? 

If so then maybe `apConditions` might be better?
  
if it's not just `apConditions` then calling it `previousLicenceConditions` might be more meaningful?

Reply:
refactored the code so removed extra variable